composer.json

```diff
@@ -18,8 +18,8 @@
         "php": "^7.4|^8.0",
         "stripe/stripe-php": "^7.75",
         "tipoff/authorization": "^2.5.5",
-        "tipoff/support": "^1.6.3",
-        "tipoff/vouchers": "^2.2.2"
```

Comment:
Dropping vouchers dependency is intentional.  Its all handled via interfaces now.